server/src/main/java/org/elasticsearch/common/geo/Extent.java

```diff
@@ -79,8 +77,6 @@ public void reset(int top, int bottom, int negLeft, int negRight, int posLeft, i
      * @param topRightY   the top-right y-coordinate
      */
     public void addRectangle(int bottomLeftX, int bottomLeftY, int topRightX, int topRightY) {
-        assert bottomLeftX <= topRightX;
-        assert bottomLeftY <= topRightY;
         this.bottom = Math.min(this.bottom, bottomLeftY);
```

Comment:
I think this assertions are actually important, why did you remove them?

Reply:
the geocentroid aggregator tests were failing, maybe I was too quick to do this and it is a problem with the test.

Reply:
I think the extent will only work if this assertions are true